tests/tests.cpp

```diff
@@ -4,12 +4,14 @@
   #include <mingw.condition_variable.h>
   #include <mingw.shared_mutex.h>
   #include <mingw.future.h>
+  #include <mingw.latch.h>
```

Comment:
Inclusion of and tests for `latch` should be gated behind the necessary `__cplusplus` version. In particular, the implemented `latch` requires a C++20 compiler.

Without this conditional inclusion and compilation, the test suite will no longer function on older compilers or standard versions.